### PR TITLE
fix(textproto): use 'pbtxt' filetype

### PR DESCRIPTION
**Pull request by ambroisie**
This is the actual name of the detected filetype.

### CODE REVIEW

**Comment by ambroisie** [comment]
Once https://github.com/vim/vim/pull/14463 is merged into NeoVim, we can even remove the `vim.filetype.add` call.

File: scripts/minimal_init.lua [hunk 1]
```diff
@@ -8,7 +8,7 @@ vim.filetype.add {
     cmm = "t32",
     hurl = "hurl",
     ncl = "nickel",
-    textproto = "textproto",
+    textproto = "pbtxt",
```
Comment (clason) [inline scripts/minimal_init.lua:11]:
```suggestion
```
If `pbtxt` is _the_ filetype, then this is not needed.

Comment (ambroisie) [reply scripts/minimal_init.lua:11]: The test files are using `*.textproto` as their extension.

Comment (ambroisie) [reply scripts/minimal_init.lua:11]: I thought #5402 had added test files using the `*.textproto`  extension, I was mistaken.

Comment (clason) [reply scripts/minimal_init.lua:11]: Even if so, these could be changed ;)

**Pull request merged**